editor/src/clj/editor/defold_project.clj

```diff
@@ -447,12 +447,6 @@
                  (not (Boolean/getBoolean "defold.tests")))
         (log/info :message "Some files were migrated and will be saved in an updated format." :migrated-proj-paths migrated-proj-paths)))))
 
-(defn connect-if-output [src-type src tgt connections]
-  (let [outputs (g/output-labels src-type)]
-    (for [[src-label tgt-label] connections
-          :when (contains? outputs src-label)]
-      (g/connect src src-label tgt tgt-label))))
-
```

Comment:
Moved to `editor.graph-util/connect-existing-outputs`. I didn't like the original name since it implied a single output.